### ISSUE TITLE
high loss value

### ISSUE THREAD

**Issue by DonaldKam** [ifzhang/FairMOT#117]
Thanks for the great job. I trained on MOT17 with hrnet w18, but reproduced no good results and the loss value converged to approximately 3.5.

**Comment by ifzhang** [comment]
I have that loss too. HRNet-W18 seems performing well on the validation set. But I test it on MOT17 test set and find it gets worse results than DLA34.
